fdbserver/GrvProxyServer.actor.cpp

```diff
@@ -543,6 +547,7 @@ ACTOR Future<GetReadVersionReply> getLiveCommittedVersion(SpanID parentSpan,
 	GetRawCommittedVersionReply repFromMaster = wait(replyFromMasterFuture);
 	grvProxyData->minKnownCommittedVersion =
 	    std::max(grvProxyData->minKnownCommittedVersion, repFromMaster.minKnownCommittedVersion);
+	grvProxyData->ssVersionVectorCache = repFromMaster.ssVersionVector;
```

Comment:
Is this cache really needed? It seems `GetReadVersionReply` can just use `repFromMaster.ssVersionVector` without this assignment.

Reply:
It is not needed right now but we may need it when we implement version vector related optimizations (for example, the version vector changes that the proxy will receive from the sequencer could be different from the version vector changes that the proxy will need to send to the client).